packages/lesswrong/components/ea-forum/SmallpoxBanner.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { createStyles } from '@material-ui/core/styles'
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { SECTION_WIDTH } from '../common/SingleColumnSection';
+import { cloudinaryCloudNameSetting } from '../../lib/publicSettings';
+import { Link } from '../../lib/reactRouterWrapper';
+
+const bannerHeight = 250
+const mobileImageId = 'Banner/Smallpox_Eradication_Day-17.png'
+const desktopImageId = 'Banner/Smallpox_Eradication_Day-16.png'
+const featuredPost = '/posts/jk7A3NMdbxp65kcJJ/500-million-but-not-a-single-one-more'
+
+const mobileImage = `https://res.cloudinary.com/${cloudinaryCloudNameSetting.get()}/image/upload/w_${SECTION_WIDTH},h_${bannerHeight}/${mobileImageId}`
+const desktopImage = `https://res.cloudinary.com/${cloudinaryCloudNameSetting.get()}/image/upload/w_${SECTION_WIDTH},h_${bannerHeight}/${desktopImageId}`
+
+const styles = createStyles((theme: ThemeType): JssStyles => ({
+  link: {
+    '&:hover': {
+      opacity: 'unset'
+    }
+  },
+  image: {
+    objectFit: 'cover'
+  }
+}))
+
+const SmallpoxBanner = ({ classes }) => {
+  const { SingleColumnSection } = Components
+  
+  return <SingleColumnSection>
+    <Link to={featuredPost} className={classes.link}>
+      <picture>
+        {/* 960px is the md boundary */}
+        <source media="(max-width: 959px)" srcSet={mobileImage} />
```

Comment:
Not sure if your version of material supports this, but if it does, you could also get `theme` object with the [`useTheme()` hook](https://material-ui.com/styles/api/#usetheme-theme), and then use `theme.breakpoints.values.sm` to get the value.

Relatedly (and very minor), you might also want to express it as `960 - 0.05px`/`959.95` to account for devices with 2x resolution that can resolve sub-pixel values.

Reply:
There's no `useTheme()` on this version of MUI.